libraries/testing/CMakeLists.txt

```diff
@@ -4,10 +4,13 @@ configure_file(contracts.hpp.in include/testing_contracts/contracts.hpp ESCAPE_Q
 add_library(eosio_testing_contracts INTERFACE)
 target_include_directories(eosio_testing_contracts INTERFACE ${CMAKE_CURRENT_BINARY_DIR}/include/testing_contracts)
 
+configure_file(contracts.cpp.in contracts.cpp ESCAPE_QUOTES)
```

Comment:
There is a defect here. `contracts.cpp` depends on the various wasm/abi files it embeds and the build system is oblivious to that. Unfortunately attaching proper dependencies is going to require some reorganization due to these wasm/abi files being placed in to `CMAKE_BINARY_DIR` via `unittests/contracts`, which is currently executed after `libraries/testing`. I don't have a low touch fix for the reorg required to properly fix.